cos-csi-mounter/server/utils.go

```diff
@@ -0,0 +1,102 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+// MountRequest ...
+type MountRequest struct {
+	Path    string          `json:"path"`
+	Bucket  string          `json:"bucket"`
+	Mounter string          `json:"mounter"`
+	Args    json.RawMessage `json:"args"`
+}
+
+var (
+	// Directories where bucket can be mounted
+	safeMountDirs = []string{"/var/data/kubelet/pods", "/var/lib/kubelet/pods"}
+	// Directories where s3fs/rclone configuration files need to be present
+	safeMounterConfigDir = "/var/lib/cos-csi"
+)
+
+// MounterArgs ...
+type MounterArgs interface {
+	Validate(path string) error
+	PopulateArgsSlice(bucket, path string) ([]string, error)
+}
+
+func strictDecodeForUnknownFields(data json.RawMessage, v interface{}) error {
+	dec := json.NewDecoder(bytes.NewReader(data))
+	dec.DisallowUnknownFields()
+	return dec.Decode(v)
+}
+
+func pathValidator(targetPath string) error {
+	absPath, err := filepath.Abs(targetPath)
+	if err != nil {
+		return fmt.Errorf("failed to resolve absolute mount path: %v", err)
+	}
+	if !(strings.HasPrefix(absPath, safeMountDirs[0]) || strings.HasPrefix(absPath, safeMountDirs[1])) {
```

Comment:
mount path for both s3fs and rclone is `/var/data/kubelet/pods` as per the tests we have run so far. 

It will be good add comments on in which scenario we use `/var/lib/kubelet/pods`  path